examples/withdraw_from_derive.py

```diff
@@ -2,219 +2,43 @@
 Example of bridging funds from a Derive smart contract funding account to BASE
 """
 
-import json
 import os
 
+import click
 from dotenv import load_dotenv
-from eth_account import Account
-from web3 import Web3
-from web3.contract import Contract
 
-from derive_client.bridge.constants import MSG_GAS_LIMIT, PAYLOAD_SIZE, TARGET_SPEED
-from derive_client.bridge.enums import ChainID, Currency, DRPCEndPoints, TxStatus
+from derive_client.bridge.enums import ChainID, Currency
 from derive_client.bridge.models import Address
-from derive_client.bridge.transaction import sign_and_send_tx
-from derive_client.bridge.utils import (
-    estimate_fees,
-    exp_backoff_retry,
-    get_contract,
-    get_erc20_contract,
-    get_prod_lyra_addresses,
-    get_repo_root,
-)
 from derive_client.derive import DeriveClient
 from tests.conftest import Environment
 
-CONTROLLER_ABI_PATH = get_repo_root() / "data" / "controller.json"
-DEPOSIT_HOOK_ABI_PATH = get_repo_root() / "data" / "deposit_hook.json"
-LIGHT_ACCOUNT_ABI_PATH = get_repo_root() / "data" / "light_account.json"
-L1_CHUG_SPLASH_PROXY_ABI_PATH = get_repo_root() / "data" / "l1_chug_splash_proxy.json"
-L1_STANDARD_BRIDGE_ABI_PATH = get_repo_root() / "data" / "l1_standard_bridge.json"
-WITHDRAW_WRAPPER_V2 = get_repo_root() / "data" / "withdraw_wrapper_v2.json"
+ChainChoice = click.Choice([f"{c.name}" for c in ChainID])
```

Comment:
dont need the fstring here, a generator works no?

Reply:
changed like this in upcoming PR
```
-ChainChoice = click.Choice([f"{c.name}" for c in ChainID])
-CurrencyChoice = click.Choice(map(str, Currency))
+ChainChoice = click.Choice(c.name for c in ChainID)
+CurrencyChoice = click.Choice(c.name for c in Currency)
```